### PR TITLE
Adding perks for historical characters

### PR DESCRIPTION
**Pull request by MemeWizard**
## Changelog:
- Now historical characters get their historical perks and lifestyle traits on gamestart.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
- Check historical characters perks and traits 
  - **Any random perks that should have been allocated on gamestart were turned off for testing convenience** 
  - Historical characters should get at least minimal amount of perks required for their historical trait
    - It means that, for example, they shouldn't have level 5 warlock trait but have only 15 fel magic perks (lvl5 requires 16)

### CODE REVIEW

File: common/scripted_effects/wc_history_effects.txt [hunk 1]
```diff
@@ -21,4 +21,242 @@ add_patron_trait_character_effect = {
...
+		if = {
+			limit = { has_variable = wc_rogue_lifestyle_additional_perks_variable }
+			change_variable = { name = wc_physical_lifestyle_additional_perks_variable add = var:wc_rogue_lifestyle_additional_perks_variable }
+		}
+		if = {
+			limit = { has_variable = wc_hunter_lifestyle_additional_perks_variable }
+			change_variable = { name = wc_physical_lifestyle_additional_perks_variable add = var:wc_hunter_lifestyle_additional_perks_variable }
+		}
+
+		# while = {
```
Comment (zumbak04) [inline common/scripted_effects/wc_history_effects.txt:203]: It's commented.

Vek'nilash is a shadow priest though supposed to be a warrior.
<details>
<summary>Click to expand</summary>

image
image

</details>

File: common/scripted_effects/wc_history_effects.txt [hunk 2]
```diff
@@ -21,4 +21,242 @@ add_patron_trait_character_effect = {
...
+			add_perk = $TYPE$_tree_$TREE$_perk_9
+		}
+		else = { add_$TYPE$_lifestyle_perk_points = 1 }
+
+	# }
+}
+
+add_perks_in_nonelemental_MAGIC_lifestyle_effect = {
+	if = {
+		limit = { has_variable = wc_$MAGIC$_magic_lifestyle_additional_perks_variable }
```
Comment (zumbak04) [inline common/scripted_effects/wc_history_effects.txt:102]: If you don't need the variable for future use, clear it.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/scripted_effects/wc_history_effects.txt (hunk 1 above)
Comment (MemeWizard) [reply common/scripted_effects/wc_history_effects.txt]: > It's commented.

Added physical perks

> Vek'nilash is a shadow priest though supposed to be a warrior.

It should be fixed on the later branch that fixes auto selection weight. I can't fix it on this PR. So, please, only rate giving perks to characters from history by event and not from automatic perk allocation by the game.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by TheGroxEmpire — COMMENTED** [review]
- Immensely and Moderately looked really similar at glance, same with Master and Adept. Should maybe try to have the 2 stars behind a lot more pronounced:
<details>

image
image

image
image

</details>

- I'm not sure what defined a character as historical that they would be eligible for starting perks. But there are plenty of lore notable character that are missing the historical traits:
1. The dragon aspects (All Bookmark).
2. Anub'Rekhan (583).
3. Gelbin (All bookmark).
4. Mandokir (583).
5. Loken (All bookmark).
6. Jin'do (All bookmark).

- Thrall being a shaman had so many trait that he had a lot of knowledge overflow at the start of the game. He'd never be able to earn any more perk:
<details>

image

</details>

- Cenarius got downgraded from Master Druid in 583 to Expert Druid in 603:
<details>

image
</details>

- Remulos should be a Master Druid instead of Expert Druid:
<details>

image
</details>

**Comment by MemeWizard** [comment]
> Should maybe try to have the 2 stars behind a lot more pronounced:

I changed traits icons on a different branch in 7893e75d92c67ccfbb2deeb21e6e98ccd91147bc, but idk when it will be merged

**Review by TheGroxEmpire — APPROVED** [review]
My suggestions were applied and confirmed.

**Comment by MemeWizard** [comment]
@zumbak04 @TheGroxEmpire thanks for reviews <3

**Pull request merged**